konnectivity-client/go.mod

```diff
@@ -5,18 +5,19 @@ go 1.17
 // Prefer to keep requirements compatible with the oldest supported
 // k/k minor version, to prevent client backport issues.
 require (
-	github.com/golang/protobuf v1.4.3
+	github.com/golang/protobuf v1.5.2
```

Comment:
This would likely prevent backporting konnectivity-client to earlier k/k release branch.

Is there any CVE impact if we simply leave konnectivity-client as is?